test-gen-plugin/src/main/java/com/dylibso/chicory/maven/JavaTestGen.java

```diff
@@ -115,12 +102,7 @@ public CompilationUnit generate(
         var testClass = cu.addClass(testName);
         if (ordered) {
             testClass.addSingleMemberAnnotation(
-                    "TestMethodOrder",
-                    new ClassExpr(
-                            JAVA_PARSER
-                                    .parseType("MethodOrderer.OrderAnnotation")
-                                    .getResult()
-                                    .get()));
```

Comment:
For the records, the JavaParser issue was in this type lookup.
The rest of the fixes are to tackle other failures that I have observed.

Reply:
Nice catch!